tests/src/3d/testqgs3drendering.cpp

```diff
@@ -1408,5 +1469,155 @@ void TestQgs3DRendering::testFilteredExtrudedPolygons()
+  QMouseEvent mouseEvent( QEvent::MouseButtonPress, QPointF( midPos.x(), midPos.y() ),
+                          Qt::MiddleButton, Qt::MiddleButton, Qt::NoModifier );
+  testCam->superOnMousePressed( new Qt3DInput::QMouseEvent( mouseEvent ) );
+
+  // First wheel action
+  QWheelEvent wheelEvent( midPos, midPos, QPoint(), QPoint( 0, 120 ),
+                          Qt::NoButton, Qt::NoModifier, Qt::NoScrollPhase,
+                          false, Qt::MouseEventSynthesizedByApplication );
+  testCam->superOnWheel( new Qt3DInput::QWheelEvent( wheelEvent ) );
+  qDebug() << "Checking depth after onWheel called" ;
+  QCOMPARE( testCam->cumulatedWheelY(), wheelEvent.angleDelta().y() * 5 );
+  QCOMPARE( testCam->cameraBeforeZoom()->viewCenter(), testCam->cameraPose()->centerPoint().toVector3D() );
+
+  depthImage = Qgs3DUtils::captureSceneDepthBuffer( engine, scene );
+  grayImage = convertDepthImageToGray16Image( depthImage );
+  QVERIFY( renderCheck( "depth_wheel_action_1", grayImage, 550 ) );
+
+  scene->cameraController()->depthBufferCaptured( depthImage );
+
+  QGSCOMPARENEARVECTOR3D( testCam->zoomPoint(), QVector3D( -32.7, 224.6, 185.5 ), 1.0 );
+  QGSCOMPARENEARVECTOR3D( testCam->cameraPose()->centerPoint(), QVector3D( -32.7, 224.6, 185.5 ), 1.0 );
+  QGSCOMPARENEAR( testCam->cameraPose()->distanceFromCenterPoint(), 955.4, 1.0 );
+  QCOMPARE( testCam->cumulatedWheelY(), 0 );
+
+  // Second wheel action
+  QWheelEvent wheelEvent2( midPos, midPos, QPoint(), QPoint( 0, 120 ),
+                           Qt::NoButton, Qt::NoModifier, Qt::NoScrollPhase,
+                           false, Qt::MouseEventSynthesizedByApplication );
+  testCam->superOnWheel( new Qt3DInput::QWheelEvent( wheelEvent2 ) );
+  qDebug() << "Checking depth after onWheel called (bis)" ;
+  QCOMPARE( testCam->cumulatedWheelY(), wheelEvent2.angleDelta().y() * 5 );
+  QCOMPARE( testCam->cameraBeforeZoom()->viewCenter(), testCam->cameraPose()->centerPoint().toVector3D() );
+
+  depthImage = Qgs3DUtils::captureSceneDepthBuffer( engine, scene );
+  grayImage = convertDepthImageToGray16Image( depthImage );
+  QVERIFY( renderCheck( "depth_wheel_action_2", grayImage, 550 ) );
+
+  scene->cameraController()->depthBufferCaptured( depthImage );
+
+  QGSCOMPARENEARVECTOR3D( testCam->zoomPoint(), QVector3D( -32.5, 223.5, 184.7 ), 1.0 );
+  QGSCOMPARENEARVECTOR3D( testCam->cameraPose()->centerPoint(), QVector3D( -32.5, 223.5, 184.7 ), 1.0 );
+  QGSCOMPARENEAR( testCam->cameraPose()->distanceFromCenterPoint(), 757.4, 1.0 );
+  QCOMPARE( testCam->cumulatedWheelY(), 0 );
+
+  // Third wheel action
+  QWheelEvent wheelEvent3( midPos, midPos, QPoint( ), QPoint( 0, 480 ),
+                           Qt::NoButton, Qt::NoModifier, Qt::NoScrollPhase,
+                           false, Qt::MouseEventSynthesizedByApplication );
+  testCam->superOnWheel( new Qt3DInput::QWheelEvent( wheelEvent3 ) );
+  qDebug() << "Checking depth after onWheel called (ter)" ;
```

Comment:
remove this line or change it to a proper message.

Reply:
done!